sql/org_mozilla_firefox_derived/incline_executive_v1/query.sql

```diff
@@ -1,37 +1,29 @@
 CREATE TEMP FUNCTION bucket_manufacturer(manufacturer STRING) AS (
-  IF(
-    manufacturer IN (
-      'samsung',
-      'huawei',
-      'xiaomi',
-      'lge',
-      'motorola',
-      'sony',
-      'google',
-      'oppo',
-      'oneplus'
-    ),
-    manufacturer,
-    'Other'
-  )
+  IF(LOWER(manufacturer) IN ('samsung', 'huawei'), LOWER(manufacturer), 'Other')
 );
 
 CREATE TEMP FUNCTION bucket_country(country STRING) AS (
   IF(
     country IN ('US', 'CA', 'DE', 'IN', 'FR', 'CN', 'IR', 'BR', 'IE', 'GB', 'ID'),
```

Comment:
The established definition of tier 1 used in 2019 KPIs is (US, CA, DE, FR, GB). IT looks like we're including additional countries as tier 1 here.

Reply:
Hmm, this came from the incline team meeting. Let's check in with Emily.